internal/resolution/variable_sources/entity/bundle_entity.go

```diff
@@ -212,6 +203,19 @@ func (b *BundleEntity) loadChannelProperties() error {
 	return nil
 }
 
+func (b *BundleEntity) loadBundlePath() error {
+	b.mu.Lock()
+	defer b.mu.Unlock()
+	if b.bundlePath == "" {
+		bundlePath, err := loadFromEntity[string](b.Entity, PropertyBundlePath, required)
+		if err != nil {
+			return fmt.Errorf("error determining bundle path for entity '%s': %w", b.ID, err)
+		}
+		b.bundlePath = bundlePath
+	}
+	return nil
+}
+
 func loadFromEntity[T interface{}](entity *input.Entity, propertyName string, required propertyRequirement) (T, error) {
```

Comment:
Not related to this PR, but it would be awesome if you could change this to `[T any]` =D